persistence/sql/migrations/20241609000001000000_device_flow.postgres.down.sql

```diff
@@ -0,0 +1,17 @@
+ALTER TABLE hydra_oauth2_device_auth_codes DROP CONSTRAINT IF EXISTS hydra_oauth2_device_auth_codes_challenge_id_fk;
+ALTER TABLE hydra_oauth2_device_auth_codes DROP CONSTRAINT IF EXISTS hydra_oauth2_device_auth_codes_client_id_fk;
+ALTER TABLE hydra_oauth2_device_auth_codes DROP CONSTRAINT IF EXISTS hydra_oauth2_device_auth_codes_nid_fk_idx;
```

Comment:
IIRC this would already be covered by `DROP TABLE ...`, right?

Reply:
good catch, I removed those lines and now we again have one "down" migrations for all database types